Wrappers.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "CoffeeMachine.cpp"
+#include "Fridge.cpp"
+#include "Amplifier.cpp"
+#include "Kitchen.cpp"
+#include "House.cpp"
+
+struct CoffeeMachineWrapper
+{
+    CoffeeMachineWrapper(CoffeeMachine* ptr) : coffPtr(ptr) {}
+    ~CoffeeMachineWrapper()
+    {
+        delete coffPtr;
+    }
+
+    CoffeeMachine* coffPtr = nullptr;
+
+};
+
+struct FridgeWrapper
+{
+    FridgeWrapper(Fridge* ptr) : fridgePtr(ptr) {}
+    ~FridgeWrapper()
+    {
+        delete fridgePtr;
+    }
+
+    Fridge* fridgePtr = nullptr;
+};
+
+struct AmpWrapper
+{
+    AmpWrapper(Amplifier* ptr) : ampPtr(ptr) {}
+    ~AmpWrapper()
+    {
+        delete ampPtr;
+    }
+
+    Amplifier* ampPtr = nullptr;
+
+};
+
+struct KitchenWrapper
+{
+    KitchenWrapper(Kitchen* ptr) : kitPtr(ptr) {}
+    ~KitchenWrapper()
+    {
+        delete kitPtr;
+    }
+
+    Kitchen* kitPtr = nullptr;
+};
+
+struct HouseWrapper
+{
+    HouseWrapper(House* ptr) : housePtr(ptr) {}
+    ~HouseWrapper()
+    {
+        delete housePtr;
+    }
+
+    House* housePtr = nullptr;
+};
```

Comment:
```
 5) NO IN-CLASS IMPLEMENTATION ALLOWED.
         the only exception is the existing Atomic.h and LeakedObjectDetector.h
        Chances are extremely high that you implemented your 
        wrapper class functions in-class, because that is what 
        everyone does after watching the video where I implemented 
        them in-class.
```
- [ ] revise